api/envoy/config/api_listener/v2/api_listener.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package envoy.config.listener.v2;
```

Comment:
Nit: also change the directory to `api/envoy/config/listener/v2`. LGTM once that is done.

Reply:
Done.